cargo-shuttle/src/lib.rs

```diff
@@ -101,13 +101,16 @@ impl Shuttle {
         args: ShuttleArgs,
         provided_path_to_init: bool,
     ) -> Result<CommandOutcome> {
-        trace!("running local client");
-
-        if args.api_url.as_ref().is_some_and(|s| s.ends_with('/')) {
-            println!(
-                "WARNING: API URL is probably incorrect. Ends with '/': {}",
-                args.api_url.clone().unwrap()
-            );
+        if let Some(ref s) = args.api_url {
+            if s != API_URL_DEFAULT {
+                eprintln!("INFO: Targetting non-standard API: {}", s);
```

Comment:
Can we print at `stdout`? I would keep `stderr` exclusively for errors and this doesn't seem to be an error.

Reply:
AFAIK the convention is to have only the main output on stdout and everything else on stderr. (think curl progress bar etc)

Reply:
I'm leaning towards println here too, since this is more of a hint. The docs for eprintln! says to only use it for errors and progress messages. But I could be wrong here.